src/test/java/org/broadinstitute/consent/http/service/UserServiceTest.java

```diff
@@ -111,7 +111,8 @@ public void testUpdateUserFieldsById() {
             fields.setSelectedSigningOfficialId(1);
             fields.setSuggestedSigningOfficial(RandomStringUtils.random(10, true, false));
             fields.setSuggestedInstitution(RandomStringUtils.random(10, true, false));
-            assertEquals(3, fields.buildUserProperties(user.getUserId()).size());
+            fields.setDaaAcceptance(true);
+            assertEquals(4, fields.buildUserProperties(user.getUserId()).size());
```

Comment:
The ticket calls for UserService.updateUserFieldsById() and UserResource.updateSelf() to be updated but I see that both of those end eventually call userUpdateFields.buildUserProperties which his where daaAcceptance is set, so that makes sense. Adding a test to check for daaAcceptance in  UserResourceTest would be redundant, right?

Reply:
Correct. There's no reason to re-test Service class logic in the Resource class test(s).